source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst

```diff
@@ -242,8 +242,8 @@ MCU GPIO wakeup can only be tested when
 overlay is loaded. Please refer to :ref:`How to enable DT overlays<howto_dt_overlays>` for more details.
 
 Once the system has entered Deep Sleep or MCU Only mode as shown in the
-:ref:`LPM section<lpm_modes>`, wakeup from MCU GPIO0_16 can be triggered
-by grounding Pin 11 on J8 MCU Header.
+:ref:`LPM section<lpm_modes>`, wakeup from MCU_SPI0_D1 can be triggered
+by grounding Pin 4 on J8 MCU Header.
```

Comment:
Isn't this device specific? This file is included by AM62 AM62A and AM62P configs. It should `ifconfig`ed out for each platform if you want to mention the specific pin.

Reply:
this is not device specific to AM62 AM62A and AM62P. They all use the same overlay to create this wakeup source, so pin 4 is the correct pin for each of them